File: ios/stereoCamera/stereoCamera/Comm/Commands/Handshake.swift
//
//  HandshakeTimeout.swift
//  stereoCamera
//
//  Created by hallmarklabs on 6/25/18.
//  Copyright © 2018 cody. All rights reserved.
//

import Foundation

class Handshake : Command
{    
    override init()
    {
        super.init()
        cmdtype = CommandTypes.HANDSHAKE
        expectsResponse = true
    }
}

File: android/app/src/main/java/com/munger/stereocamera/ip/command/commands/Handshake.java
package com.munger.stereocamera.ip.command.commands;

import com.munger.stereocamera.ip.command.Command;

public class Handshake extends Command
{
	public Handshake()
	{
		super();

		cmdtype = Type.HANDSHAKE;
		expectsResponse = true;
	}
}
